.github/ISSUE_TEMPLATE/sync_request.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Upstream Autoscaler Sync 
+about: Sync with Upstream Autoscaler
+labels: kind/enhancement
+title: Issue for - Sync with Upstream v1.2x
+---
+
+## What would you like to be added
+Gardener autoscaler should be synced with [Kubernetes Autoscaler 1.2x](https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.2x.y)
+
+## Why is this needed
+
+* To keep the fork in sync with upstream.
+* To ensure the shoot control planes are using respective version of CA for a given K8S version
+
+## Steps
+
+NOTE: replace `1.2x.0` with the specific autoscaler upstream version that you are trying to sync to. Replace `user` with your github user.
+
+- [ ] Fork `github.com/gardener/autoscaler`  into your github account say `user`. 
```

Comment:
we should forward the user to the sync steps in [FAQ](https://github.com/gardener/autoscaler/blob/machine-controller-manager-provider/cluster-autoscaler/FAQ.md#how-do-i-sync-gardener-autoscaler-with-an-upstream-autoscaler-minor-release) and update the FAQ steps with what you have added here.
That way there is just one point of update/reference everytime.

Reply:
moved steps to FAQ and linked issue to FAQ